.env.example

```diff
@@ -0,0 +1,5 @@
+GA_TRACKING_ID=
+SQUIDEX_CLIENT_SECRET=
```

Comment:
add these to the readme with explanations of what they are and where to get them, and any defaults.